vertx-gateway/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM adoptopenjdk/openjdk11:alpine-jre
+
+MAINTAINER florian.volk@scheer-group.com
+EXPOSE 8080 8081
+
+ARG APIMAN_VERSION
+
+RUN addgroup -S apiman && adduser -S apiman -G apiman && \
+	mkdir -p /opt/apiman/logs && \
+	chown -R apiman:apiman /home/apiman /opt/apiman/logs && \
+    apk update && apk add curl unzip && rm -rf /var/cache/apk/*
+
+RUN curl -s -L https://github.com/Apiman/apiman/releases/download/$APIMAN_VERSION/apiman-distro-vertx-$APIMAN_VERSION.zip -o /tmp/apiman-distro-vertx-${APIMAN_VERSION}.zip && \
+	unzip /tmp/apiman-distro-vertx-${APIMAN_VERSION}.zip -d /tmp && \
```

Comment:
Is it necessary to unzip into /tmp and move everything afterwards into /opt/apiman?
Otherwise we could unzip directly into /opt/apiman

Reply:
Yes, that is because the zip contains a folder with a version number in its name.
So it's easier to just move only the content to the correct place. 

We should change that in the overlay then directly at some point.